phonebook/main_functions.py

```diff
@@ -16,55 +23,61 @@ def load_data_from_file(users, filepath):
         users.append(new_user)
 
 
-def add_to_file(filepath, user):
-    """Add new user to file.txt"""
-    file = open(filepath, "a")
-    file.writelines(user.to_string() + "\n")
-    file.close()
-
-
-def add_new_user(filepath, users):
+def get_data_from_user():
     name = input("Enter user name: ")
     surname = input("Enter a surname: ")
     phone_number = input("Enter user phone number: ")
+    user = {
+        "name": name,
+        "surname": surname,
+        "phone_number": phone_number,
+    }
+    return user
+
+
+def add_new_user(users, user_as_dictionary, filepath):
     creation_date = datetime.now()
-    new_user = User(name, surname, phone_number, creation_date)
+    new_user = User(user_as_dictionary["name"], user_as_dictionary["surname"], user_as_dictionary["phone_number"], creation_date)
     users.append(new_user)
-    add_to_file(filepath, new_user)
+    return new_user
 
 
-def show_all_users(users):
-    for user in users:
-        print(user.to_string())
+def add_to_file(filepath, user):
+    """Add new user to file.txt"""
+    file = open(filepath, "a")
+    file.writelines(user.to_string() + "\n")
+    file.close()
 
 
-"""def show_user(users):
-    searching_filter = input("Enter searching filter:")
+def show_users(users):
     for user in users:
-        print(user)
-"""
+        print(user.to_string())
 
 
-def search_user_by_name(users):
+def get_searching_name():
```

Comment:
get_searching_name is not  named consequently ( get_data_from_user )

Reply:
changed into : get_data_to_create_new_user
get_data_to_search_by_name/surname